datalad_container/extractors/tests/test_metalad_container.py

```diff
@@ -64,13 +44,13 @@ def test__container_inspect_valid(singularity_test_image):
     expected_result = {
         'data': {
             'attributes': {
-                'labels':{
+                'labels': {
                     'org.label-schema.build-date': 'Sat,_19_May_2018_07:06:48_+0000',
                     'org.label-schema.build-size': '62MB',
                     'org.label-schema.schema-version': '1.0',
                     'org.label-schema.usage.singularity.deffile': 'Singularity.testhelper',
                     'org.label-schema.usage.singularity.deffile.bootstrap': 'docker',
-                    'org.label-schema.usage.singularity.deffile.from': 'debian:stable-slim', 'org.label-schema.usage.singularity.version':
+                    'org.label-schema.usage.singularity.deffile.from': 'debian: stable-slim', 'org.label-schema.usage.singularity.version':
```

Comment:
looks like this broke the test...  

![image](https://github.com/datalad/datalad-container/assets/39889/edeb327a-cf64-40b2-b913-7f75963b46b4)

thus

```suggestion
                    'org.label-schema.usage.singularity.deffile.from': 'debian:stable-slim', 'org.label-schema.usage.singularity.version':
```